qt/python/mantidqt/project/test/test_projectloader.py

```diff
@@ -0,0 +1,64 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2017 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantidqt package
+#
+
+import unittest
+
+from os.path import isdir, expanduser
+from shutil import rmtree
+
+from mantid.api import AnalysisDataService as ADS
+from mantid.simpleapi import CreateSampleWorkspace
+from mantidqt.project import projectloader, projectsaver
+
+
+working_directory = expanduser("~") + "/project_loader_test"
```

Comment:
I would use a temp directory for these instead. There are functions on `os` for accessing these in a cross-platform way. I would actually also just move this to the whole line to `def setUp` and use `self.working_directory` rather than a global variable